common/autoware_path_distance_calculator/launch/path_distance_calculator.launch.xml

```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="path" default="/planning/scenario_planning/lane_driving/behavior_planning/path"/>
   <arg name="distance" default="~/distance"/>
-  <node pkg="path_distance_calculator" exec="path_distance_calculator_node" name="path_distance_calculator">
+  <node pkg="autoware_path_distance_calculator" exec="path_distance_calculator_node" name="path_distance_calculator">
```

Comment:
I think we need `autoware_` prefix for `exec="path_distance_calculator_node"` too.